presidio-analyzer/app.py

```diff
@@ -37,7 +38,7 @@ def __init__(self):
         self.logger.setLevel(os.environ.get("LOG_LEVEL", self.logger.level))
         self.app = Flask(__name__)
         self.logger.info("Starting analyzer engine")
-        self.engine = AnalyzerEngine()
```

Comment:
Do you think using AnalyzerEngineProvider from within AnalyzerEngine instantiation will be less of breaking change for people using the python package? 
@omri374

Reply:
Not sure I understand. How will the `AnalyzerEngineProvider` be used from within `AnalyzerEngine`?

Reply:
@SharonHart, you can still instantiate with the class itself, shouldn't be a breaking change

Reply:
When we will drop the recognizer registry and make the loading from the file by default it will wont it?